src/Helper/TemplateHelper.php

```diff
@@ -150,6 +151,20 @@ public function getTemplate($domain, $name)
     public function parameterize(TemplateInterface $template)
     {
         $params = [];
+        $validYesNoResponses = ['yes', 'y', 'no', 'n'];
```

Comment:
move this if at all to a Validator folder, also i rather have this to work by default if any character is pressed it should be yes, if nothing then no. Or similar.